resources/user-guides/attach-data.md

```diff
@@ -0,0 +1,113 @@
+# Using the Attach Data Rules Action
+
+The Attach Data rule action was added in the `Mobile Core v2.1.8` Launch extension. Attach Data is powerful and complex, and is recommended for users with advanced use cases.
+
+In order to understand how to use Attach Data, you will need a foundational knowledge on how Events flow in the SDK and the role of the SDK's Rules Engine.
```

Comment:
- Please reword as follows:
"To understand how to use Attach Data, you need foundational knowledge about how events flow in the SDK and the role of the SDK's Rules Engine.
(Lower cased the "R" in rules as it's referring to rules in general.)